<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>376</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>379</width>
    <height>376</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>379</width>
    <height>376</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="login_ID">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="login_PW">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="login_Button">
       <property name="text">
        <string>로그인</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>                      로고</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menulogin">
    <property name="title">
     <string>login</string>
    </property>
   </widget>
   <addaction name="menulogin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
